Automattic/wp-calypso
# WPCOM Block Editor: Remove the switch-to-classic feature

#### Changes proposed in this Pull Request

Now the Calypso editor has been deprecated the 'Switch to Classic Editor' link in the settings menu is defunct. This is still showing for users registered before December 2018 that weren't included in the deprecation email and have their editor set to Gutenberg. 

#### Testing instructions

With a user and site that meets the above criteria, or with D49284-code applied, confirm that you can see the 'Switch to Classic Editor' option in the More Options menu of Gutenberg (the three dots at the top right).

<img width="272" alt="image" src="https://user-images.githubusercontent.com/96462/92763244-3c8b7e80-f38b-11ea-8cd1-b1257efb6aeb.png">

Apply the diff associated with this patch and sandbox widgets.wp.com and on reload, the option should be removed.
